src/platform/packages/shared/response-ops/alerts-delete/components/modal.tsx

```diff
@@ -177,8 +180,12 @@ export const AlertDeleteModal = ({
 
   const { mutate: createAlertDeleteSchedule } = useAlertDeleteSchedule({
     services: { http },
-    onSuccess: () => {
-      notifications.toasts.addSuccess(translations.ALERT_DELETE_SUCCESS);
+    onSuccess: (message?: string) => {
+      if (message) {
+        notifications.toasts.addInfo(message);
```

Comment:
This message wouldn't be translated as it comes directly from the backend. What if we return a specific error code and map it in the frontend to the translated error message instead?